### PR TITLE
I30 copy paste unit test

### PR DESCRIPTION
**Pull request by zacanbrcom**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -244,3 +244,30 @@ export async function createPhysicalDocument(
     const document: vscode.TextDocument = await vscode.workspace.openTextDocument(filePath);
     await vscode.window.showTextDocument(document, { preview: false });
 }
+
+/**
+ * Create a dummy dataset.
+ * @return A dummy dataset.
+ */
+export function createDummyDataset(): Dataset {
```
Comment (shciubrcom) [inline src/utils.ts:252]: Move this function to `__tests__` folder please. It's not related to main functionality.

File: src/__tests__/CopyPasteService.test.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        expect(copyPasteService.canCopy(memberNode)).toBe(true);
+    });
+
+    test("[NEGATIVE TEST] Copy operation is not available on dataset node", () => {
+        expect(copyPasteService.canCopy(datasetNode)).toBe(false);
+    });
+
+    test("[POSITIVE TEST] Member content is stored internally ready for paste operation", () => {
+        copyPasteService.copy(host, datasetName, memberName).then(() => {
+            expect(copyPasteService.getMemberName()).toBe(memberName);
```
Comment (shciubrcom) [inline src/__tests__/CopyPasteService.test.ts]: Pls consider using await/async here

**Review by shciubrcom — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 1 above)
Comment (zacanbrcom) [reply src/utils.ts:252]: @shciubrcom seems that by default jest search for unit test everything is under __test__ folder, so in order to keep there the util service for dummy dataset I added an ignore pattern in the jest configuration, smth like this: "<rootDir>/src/__tests__/utils", is it ok for u or do u know other better way to do that?

File: src/__tests__/utils/DatasetUtils.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { Dataset } from "../../model/DSEntities";
```
Comment (shciubrcom) [inline src/__tests__/utils/DatasetUtils.ts:15]: Copyright header is missing

File: src/utils.ts (hunk 1 above)
Comment (shciubrcom) [reply src/utils.ts:252]: I think it's ok

**Review by shciubrcom — APPROVED** [review]

**Pull request merged**